### PR TITLE
LVV-T1788 to be approved

### PR DESCRIPTION
**Pull request by gabrirod86**
Hi Holger thanks for reviewing!

### CODE REVIEW

File: notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+   },
+   "outputs": [],
+   "source": [
+    "# array definition\n",
+    "\n",
+    "# bending modes scaling factors\n",
+    "amplitude = np.array((1, 0.1, 0.1, 0.05, 0.05, 0.01, 0.01, 0.01))\n",
+    "\n",
+    "# index scaling \n",
+    "idx_scaling = np.array((0, 1, 2, 3, 4 ,5 ,6 ,7))"
```
Comment (hdrass) [inline notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb]: Not sure I understand what “idx_scaling” is for. I seems unused.

File: notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+   "id": "2322d410-b397-466b-a5a4-b06d6a7d89dd",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# array definition\n",
+    "\n",
+    "# bending modes scaling factors\n",
+    "amplitude = np.array((1, 0.1, 0.1, 0.05, 0.05, 0.01, 0.01, 0.01))\n",
```
Comment (hdrass) [inline notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb]: Why do amplitudes repeat? and why are smaller amplitudes more often done?

Comment (hdrass) [reply notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb]: Amplitude has eight values, but there seem to be only have 7 groups of actuators.

**Review by hdrass — CHANGES_REQUESTED** [review]
Please have a look at my comments.

File: notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb (hunk 1 above)
Comment (gabrirod86) [reply notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb]: you are correct, it was a leftover of a draft version, we removed it

File: notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb (hunk 2 above)
Comment (gabrirod86) [reply notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb]: Comment1: these are preliminary values retrieved by the Tucson test stand during script testing. Real values to be found/confirmed tonight with the M2 cell.

Comment2: you are right, there was one scale factor in excess, we removed it.

Thanks a lot for reviewing the code!

**Review by hdrass — APPROVED** [review]
Thanks, looks good to me now.

**Pull request merged**